projects/com.oracle.truffle.llvm.nodes/src/com/oracle/truffle/llvm/nodes/intrinsics/llvm/x86/LLVMX86_64BitVAStart.java

```diff
@@ -231,6 +225,7 @@ private static LLVMRuntimeType getArgumentType(Object arg) {
         return type;
     }
 
+    @ExplodeLoop
```

Comment:
Why do we need the `@ExplodeLoop` here? There are no node arrays which the PE should explode, or are there?